resources/vbmc/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM docker.io/centos:centos8
+
+RUN dnf install -y python3 python3-requests python3-pip && \
+     curl https://raw.githubusercontent.com/openstack/tripleo-repos/master/tripleo_repos/main.py | python3 - -b master current-tripleo --no-stream && \
+     dnf update -y && \
+     dnf install -y python3-virtualbmc && \
+     dnf clean all && \
+     rm -rf /var/cache/{yum,dnf}/*
+
+# NOTE(dtantsur): work around old pyghmi in RDO; this operation will become
+# no-op once RDO is updated.
+RUN pip3 install 'pyghmi>=1.5.13'
```

Comment:
@dtantsur do you have any info if the pyghmi issue is fixed or not?

Reply:
we can remove this, the current version installed from RDO is 1.5.19

Reply:
I've removed `pyghmi` installation.